website/src/components/editor/Rule/index.js

```diff
@@ -1,30 +1,49 @@
 import clsx from 'clsx';
 import styles from './styles.module.css';
 import inputStyles from '../Input/styles.module.css';
+import * as ReactDnD from 'react-dnd';
+import * as ReactDndHtml5Backend from 'react-dnd-html5-backend';
 import {Input} from '../Input';
 import {Select} from '../Select';
-import React from 'react';
-import Link from '@docusaurus/Link';
+import React, {useCallback, useState, useMemo} from 'react';
 import {useFormContext} from 'react-hook-form';
 import 'react-sweet-progress/lib/style.css';
 import {Colors} from '../Colors';
 import 'react-datepicker/dist/react-datepicker.css';
 import PropTypes from 'prop-types';
 import {PercentagesForm} from './percentageForm';
 import {ProgressiveRollout} from './progressiveRolloutForm';
-import {TextArea} from '../TextArea';
+import {QueryBuilder} from 'react-querybuilder';
+import {QueryBuilderDnD} from '@react-querybuilder/dnd';
+import 'react-querybuilder/dist/query-builder.css';
+import './query-builder.css';
 
 Rule.propTypes = {
   variations: PropTypes.array,
   label: PropTypes.string.isRequired,
   isDefaultRule: PropTypes.bool,
 };
 export function Rule({variations, label, isDefaultRule}) {
-  const {register, watch} = useFormContext();
+  const [query, setQuery] = useState({
+    combinator: 'and',
+    rules: [{field: '', operator: '==', value: ''}],
```

Comment:
```suggestion
    rules: [{field: '', operator: 'eq', value: ''}],
```

Just to keep consistency with what we have in the `Select`.

Reply:
Sure thing! Just to clarify this is the way the query should look like?

- `env eq "test"`
- `env eq "prev" and user_id in [1, 2, 3, 4]`

Also, since I don't have a way to know whether a field should be a string or a number, should I always treat values as strings? For example, the above list would be converted like this `["1", "2", "3"]`

Reply:
I guess we can try to do this based on the type of content.
But you can probably put everything as string for now if it is easier, and we can evolve it later.